pkg/sdk/logging/api/v1beta1/fluentbit_types.go

```diff
@@ -80,13 +81,14 @@ type FluentbitSpec struct {
 	LogLevel string `json:"logLevel,omitempty" plugin:"default:info"`
 	// Set the coroutines stack size in bytes. The value must be greater than the page size of the running system. Don't set too small value (say 4096), or coroutine threads can overrun the stack buffer.
 	// Do not change the default value of this parameter unless you know what you are doing. (default: 24576)
-	CoroStackSize int32                       `json:"coroStackSize,omitempty" plugin:"default:24576"`
-	Resources     corev1.ResourceRequirements `json:"resources,omitempty"`
-	Tolerations   []corev1.Toleration         `json:"tolerations,omitempty"`
-	NodeSelector  map[string]string           `json:"nodeSelector,omitempty"`
-	Affinity      *corev1.Affinity            `json:"affinity,omitempty"`
-	Metrics       *Metrics                    `json:"metrics,omitempty"`
-	Security      *Security                   `json:"security,omitempty"`
+	CoroStackSize           int32                       `json:"coroStackSize,omitempty" plugin:"default:24576"`
+	Resources               corev1.ResourceRequirements `json:"resources,omitempty"`
+	ConfigReloaderResources corev1.ResourceRequirements `json:"configReloaderResources,omitempty"`
```

Comment:
I would consider a different structure here:
```
configHotReload: # pointer, which means hotreload is enabled if set
  image:
  resources:
```

Reply:
I replicated the pattern used for aggregators. Do you want to migrate them to this spec too?

Reply:
I would not touch that, but I also don't want to add 3 new fields to the already big fluentbit spec. I understand that consistency is a nice thing, but I would rely on the user having to read the docs in this case.

Reply:
How can I prevent kubebuilder from adding these subfields into the CRD?

Reply:
which one specifically?

Reply:
Content of the HotReload struct

Reply:
I don't think you can reference an object in a crd spec, do you have an example where you have seen that working?

Reply:
> I also don't want to add 3 new fields to the already big fluentbit spec
Any idea how I can skip these fields in the crd?

Reply:
I meant I would avoid adding 3 fields on the top level. This is all good as it is now.